Content.Shared/Actions/SharedActionsSystem.cs

```diff
@@ -679,6 +679,9 @@ public void PerformAction(EntityUid performer, ActionsComponent? component, Enti
             if (!action.RaiseOnUser && action.Container != null && !HasComp<MindComponent>(action.Container))
                 target = action.Container.Value;
 
+            if (action.RaiseOnAction)
+                target = actionId;
+
```

Comment:
This still looks like a code smell, especially when there's 2 separate bools controlling the same state.

Reply:
What we'll do for now is mark it obsolete and we'll take care of it when we do post wizard release action changes. Feel this can be settled by a list of events, but also would require more things out of actions first (like requirements).